config.yaml

```diff
@@ -3,8 +3,8 @@
 # This configuration spec is intended to be used for a single instance
 # of a Fuel indexer node or service.
 #
-# For more info on how the Fuel indexer works, read the book: https://fuellabs.github.io/fuel-indexer/master/
-# or specifically read up on these configuration options: https://fuellabs.github.io/fuel-indexer/master/getting-started/configuration.html
+# For more info on how the Fuel indexer works, read the docs: https://docs.fuel.network/docs/indexer/
+# or specifically read up on these configuration options: https://docs.fuel.network/docs/indexer/getting-started/indexer-service-infrastructure/
```

Comment:
I think we can use an anchor link to the indexer configuration section: https://docs.fuel.network/docs/indexer/getting-started/indexer-service-infrastructure/#fuel-indexer-service